wagtail/wagtailadmin/static_src/wagtailadmin/js/hallo-bootstrap.js

```diff
@@ -27,10 +27,11 @@ function makeHalloRichTextEditable(id) {
     }
 
     var closestObj = input.closest('.object');
+    var isDeepNested = input.closest('.struct-block').length;
```

Comment:
Maybe explicitly change this variable to a boolean?
It could be something like:
```javascript
var isRoot = input.closest('.struct-block').length == 0;
```
Then you can directly use `isRoot` instead of negating `isDeepNested`.